src/blocks/card.css

```diff
@@ -0,0 +1,125 @@
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--c-white);
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+}
+
+.card__image {
+  padding: 0 3px;
+  display: block;
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  padding: 0;
+  margin-top: 40px;
+  width: 166px;
+  box-sizing: border-box;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--c-main);
+}
+
+.card__code {
+  align-self: start;
+  display: block;
+  margin-top: 4px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--c-secondary);
+}
+
+.card__reviews {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+}
+
+.five-stars {
+  display: flex;
+  gap: 2px;
+}
+
+.star {
+  display: block;
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star.svg);
+}
+
+.star--active {
+  background-image: url(../images/star-active.svg);
+}
+
+.reviews-link {
+  display: flex;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  text-decoration: none;
+  color: var(--c-main);
+}
+
+.card__price-area {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+}
+
+.text {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--c-secondary);
+}
+
+.cost {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--c-main);
+}
```

Comment:
it's unnecessary BEM blocks, it should be elements of `card` block (all similar places except stars)